bindings/pyroot/pythonizations/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ if(tmva)
         ROOT/_pythonization/_tmva/_crossvalidation.py
         ROOT/_pythonization/_tmva/_dataloader.py
         ROOT/_pythonization/_tmva/_factory.py
+        ROOT/_pythonization/_tmva/_gnn.py
```

Comment:
Are we sure this belongs to `PYROOT_EXTRA_PY2_PY3_SOURCE`? Does the code in `_gnn.py` really work with Python2? Even if so, I would just restrict the new developments to Python3 compatibility only.

Reply:
made the change, thanks!